delta-api/src/main/java/io/cdap/delta/api/assessment/SourceTableDetail.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.delta.api.assessment;
+
+import java.util.List;
+
+/**
+ * Detailed information about a source table.
+ */
+public class SourceTableDetail extends TableDetail<List<ColumnDetail>> {
```

Comment:
I don't get why we need a separate class for source, seems like this class is no different than `TableDetail`, we can just create`TableDetail` right?

Reply:
the difference is that the source one represents the columns differently, since we want to use the SQL names for the column types and not the CDAP Schema types. The two classes are confusing though, so I'll just combine them into a single one that has both a List<ColumnDetail> and a Schema.

Reply:
actually, I can't combine them because there are cases when the `List<ColumnDetail>` contains types that are not supported by the source, usually because there isn't a good representation for them in CDAP Schema. Geo objects are an example of this.

I think the confusing part is that TableDetail is generic. I'm just going to remove that class